source/store/actions/CaseActions.js

```diff
@@ -12,26 +12,26 @@ export const actionTypes = {
 };
 
 export async function updateCase(
-  { user, caseId, formId, answerObject, status, currentPosition, formQuestions },
+  { user, caseId, formId, answerObject, signature, currentPosition, formQuestions },
   callback
 ) {
-  let answers = convertAnswersToArray(answerObject, formQuestions);
-
-  if (Config?.DISABLE_CLIENT_ENCRYPTION !== 'true')
-    if (status?.type === 'active:ongoing') {
-      const encryptedAnswers = await encryptWithAesKey(user, JSON.stringify(answers));
-      answers = { encryptedAnswers };
-    }
-
-  const body = {
-    statusType: status.type,
-    answers,
+  const updateCaseRequestBody = {
+    answers: convertAnswersToArray(answerObject, formQuestions),
     currentPosition,
     currentFormId: formId,
   };
 
+  if (Config?.DISABLE_CLIENT_ENCRYPTION !== 'true' && signature?.success === false) {
```

Comment:
This requires the `signature` to explicitly be `false` otherwise answers are not encrypted. I believe answers should always be encrypted unless signature is explicitly `true`. Currently with this, answers are never encrypted.